idl/cdc_generation.idl.hh

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace cdc {
+
+class stream_id {
+    bytes to_bytes();
+};
+
+class token_range_description {
+    dht::token token_range_end;
+    std::vector<cdc::stream_id> streams;
```

Comment:
How large this vector can be?

Reply:
It's the number of shards on the owner of this token range, so I guess <= 128?